_global/js/general.js

```diff
@@ -7,11 +7,22 @@ function replaceIcon(html)
   for(var match of (matchs || []))
   {
     var iconName = match.replace('{', '').replace('}', '');
+	var iconNamePieces = iconName.split(",");
```

Comment:
This shows an inconsistency between spaces and tabs again. Perhaps reconfigure your editor to use spaces instead of tabs?

Reply:
I think tabs are a better convention. If we're going to set a convention going forward - shouldn't it be tabs?